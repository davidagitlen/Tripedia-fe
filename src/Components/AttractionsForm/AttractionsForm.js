import React from 'react';
import './AttractionsForm.scss';

const AttractionsForm = () => {

  return(
    <p>AttractionsForm</p>
  )
};

export default AttractionsForm; 