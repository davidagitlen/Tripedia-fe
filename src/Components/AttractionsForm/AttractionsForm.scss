
.attraction-form__container {
  width: 100%;
  height: 32vh;
  background: blanchedalmond;
  display: flex;
  align-items: center;
  justify-content: center;
  flex-direction: column;
}
.checkbox__container {
  display: flex;
  flex-flow: row wrap;
  height: 90%;
  padding: 20px;
}
.individual-checkbox__container {
  display: flex;
  width: 50%;
  padding: 10px;
}
.attraction__checkbox {
  margin-right: 10px;
}