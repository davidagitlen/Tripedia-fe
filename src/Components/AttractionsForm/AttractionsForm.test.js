import React from 'react';
import { shallow } from 'enzyme';
import AttractionsForm from './AttractionsForm';

describe('AttractionsForm', () => {

  it('should pass a dummy test', () => {
    expect(true).toEqual(true);
  });

});