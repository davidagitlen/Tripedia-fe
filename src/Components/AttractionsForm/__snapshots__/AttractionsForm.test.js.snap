// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AttractionsForm should mock formContext and match a snapshot 1`] = `
<AttractionsForm
  collapseForm={[MockFunction]}
  defaultForm={
    Object {
      "AccommodationsForm": false,
      "AttractionsForm": false,
      "DrinksForm": false,
      "FoodForm": false,
      "ServicesForm": false,
      "StartForm": false,
      "SubmitTripForm": false,
    }
  }
  formObject={
    Object {
      "ArtGalleries": Array [],
      "DayCamps": Array [],
      "WalkingTours": Array [],
    }
  }
  openForm={[MockFunction]}
>
  <div
    className="form-closed__container"
    onClick={[Function]}
  >
    <img
      alt="attractions"
      src="attractions-pin.svg"
    />
    <p>
      - Attractions -
    </p>
  </div>
</AttractionsForm>
`;
